test/unit/system/redis_client_pool_test.rb

```diff
@@ -77,4 +77,11 @@ class System::RedisClientPoolTest < ActiveSupport::TestCase
       assert_equal 'redis-master', conf.name
     end
   end
+
+  test 'default pool is available for reuse' do
+    pool1 = System::RedisClientPool.default
```

Comment:
It's hard to test the actual config, because we can't really stub here - as the `default` is initialized before we get to the test's code - in `active_job_uniqueness` initializer.